src/main/java/slimeknights/tconstruct/common/data/TConstructRecipeProvider.java

```diff
@@ -685,6 +675,92 @@ private void addSmelteryRecipes(Consumer<IFinishedRecipe> consumer) {
+    this.addBlockCastingRecipe(consumer, TinkerFluids.moltenGlass, TinkerCommons.clearGlass, MaterialValues.VALUE_Glass);
+    CastingRecipeBuilder.tableRecipe(TinkerCommons.clearGlassPane)
+      .setFluid(new FluidStack(TinkerFluids.moltenGlass.get(), MaterialValues.VALUE_Glass * 6 / 16))
+      .addCriterion("has_item", this.hasItem(TinkerCommons.clearGlassPane.asItem()))
+      .build(consumer, location("casting/clear_glass_pane"));
+    this.addBucketCastingRecipe(consumer, TinkerFluids.moltenObsidian);
+    this.addBlockCastingRecipe(consumer, TinkerFluids.moltenObsidian, Items.OBSIDIAN, MaterialValues.VALUE_BrickBlock);
+    // Molten objects with Bucket, Block, Ingot, and Nugget forms with standard values
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenIron, Items.IRON_BLOCK, Items.IRON_INGOT, Items.IRON_NUGGET);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenGold, Items.GOLD_BLOCK, Items.GOLD_INGOT, Items.GOLD_NUGGET);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenCobalt, TinkerMaterials.cobaltBlock, TinkerMaterials.cobaltIngot, TinkerMaterials.cobaltNugget);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenArdite, TinkerMaterials.arditeBlock, TinkerMaterials.arditeIngot,  TinkerMaterials.arditeNugget);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenManyullyn, TinkerMaterials.manyullynBlock, TinkerMaterials.manyullynIngot, TinkerMaterials.manyullynNugget);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenPigIron, TinkerMaterials.pigironBlock, TinkerMaterials.pigironIngot, TinkerMaterials.pigironNugget);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenKnightslime, TinkerMaterials.knightSlimeBlock, TinkerMaterials.knightslimeIngot, TinkerMaterials.knightslimeNugget);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenCopper, TinkerMaterials.copperBlock, TinkerMaterials.copperIngot, TinkerMaterials.copperNugget);
+    this.addMoltenMineralCastingRecipe(consumer, TinkerFluids.moltenRoseGold, TinkerMaterials.roseGoldBlock, TinkerMaterials.roseGoldIngot, TinkerMaterials.roseGoldNugget);
+
+    // Smeltery Misc
+    CastingRecipeBuilder.basinRecipe(TinkerSmeltery.searedCobble.get())
+      .setFluid(new FluidStack(TinkerFluids.searedStone.get(), MaterialValues.VALUE_Ingot * 3))
+      .setCast(Tags.Items.COBBLESTONE, true)
+      .addCriterion("has_item", this.hasItem(TinkerFluids.searedStone.asItem()))
+      .build(consumer, location("casting/seared_cobble"));
+
+    CastingRecipeBuilder.basinRecipe(TinkerSmeltery.searedGlass)
+      .setFluid(new FluidStack(TinkerFluids.searedStone.get(), MaterialValues.VALUE_Ingot * 4))
+      .setCast(Tags.Items.GLASS_COLORLESS, true)
+      .addCriterion("has_item", this.hasItem(TinkerFluids.searedStone.asItem()))
+      .build(consumer, location("casting/seared_glass"));
+
+    // Two recipes for now until tagged.
```

Comment:
Two recipes? Also, we have compound ingredients for the record, to pass in multiple input options

Reply:
Actually I just missed removing the comment. it was for vanilla and clear
glass panes before you tagged them.